### PR TITLE
Adding vertx HttpClient test to http/vertx scenario

### PR DESCRIPTION
**Pull request by jcarranzan**
### Summary

(Summarize the problem solved by this PR, and how to verify it manually)

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [ ] Refactoring
- [ ] Backport
- [x] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by QuarkusQE** [comment]
Can one of the admins verify this patch?

File: http/vertx/src/test/java/io/quarkus/ts/vertx/AbstractVertxIT.java [hunk 1]
```diff
@@ -41,6 +44,20 @@ public void httpServerParsing() {
         requests().get("/hello?name=you").then().statusCode(HttpStatus.SC_OK).body("content", is("Hello, you!"));
     }
 
+    @Test
+    public void httpClient() {
```
Comment (rsvoboda) [inline http/vertx/src/test/java/io/quarkus/ts/vertx/AbstractVertxIT.java:48]:
```suggestion
    public void vertxHttpClient() {
```

File: http/vertx/src/test/java/io/quarkus/ts/vertx/AbstractVertxIT.java [hunk 2]
```diff
@@ -41,6 +44,20 @@ public void httpServerParsing() {
...
+        HttpClient httpClient = Vertx.vertx().createHttpClient();
+        httpClient.request(HttpMethod.GET, 1101, "localhost", "/hello")
```
Comment (rsvoboda) [inline http/vertx/src/test/java/io/quarkus/ts/vertx/AbstractVertxIT.java:50]: Hard-coded localhost and port is not good, find a way how to get these from the code, consult the framework wiki

**Pull request merged**